panel/src/components/Dialogs/index.js

```diff
@@ -38,6 +39,7 @@ export default {
 		app.component("k-form-dialog", FormDialog);
 		app.component("k-license-dialog", LicenseDialog);
 		app.component("k-link-dialog", LinkDialog);
+		app.component("k-lock-dialog", LockDialog);
```

Comment:
`Suggestion:` As this sounds ambitious (could be a dialog to lock a page etc.) how about `LockedContentDialog` or `LockAlertDialog`?

Reply:
I like LockAlertDialog